src/data/showcases.js

```diff
@@ -1198,6 +1198,14 @@ export const Showcases = [
     source: null,
     tags: ["wallet", "nftsupport"],
   },
+  {
+    title: "NOWNodes",
+    description: "Get instant access to Cardano RPC nodes, indexer, and WebSocket connection. Start seamless integration on a FREE tariff plan. You can focus on growing your web3 products with us.",
+    preview: require("./showcase/nownodes.png"),
+    website: "https://nownodes.io",
+    source: null,
+    tags: ["daotool", "explorer"],
```

Comment:
```suggestion
    tags: ["explorer"],
```
I don't see how this fits the `daotool` category so @AnnaNodes perhaps you could explain here why you included it: or merge this suggestion to remove this keyword.